### PR TITLE
[MIG] Migrating crm_location to v11

### PR DESCRIPTION
**Pull request by hprasetyou**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please preserve commit history following technical method explained in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0

File: crm_location/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    'category': 'Customer Relationship Management',
+    'version': '11.0.1.0.0',
+    'depends': [
+        'crm',
+        'base_location',
+    ],
+    'data': ['views/crm_lead_view.xml'],
+    'author': 'Tecnativa, '
+              'Odoo Community Association (OCA)',
+    'website': 'https://www.tecnativa.com',
```
Comment (MeetKD) [inline crm_location/__manifest__.py:16]: Change website key https://github.com/OCA/crm

File: crm_location/views/crm_lead_view.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    <field name="name">Add city completion helper field</field>
+    <field name="model">crm.lead</field>
+    <field name="inherit_id" ref="crm.crm_case_form_view_oppor" />
+    <field name="arch" type="xml">
+        <field name="city" position="before">
+            <field name="location_id"
+                options="{'create_name_field': 'city'}"
+                colspan="4"
+                placeholder="City completion"/>
+        </field>
```
Comment (MeetKD) [inline crm_location/views/crm_lead_view.xml:28]: Minor suggestion placeholder should be select city/location/zip.

**Review by MeetKD — CHANGES_REQUESTED** [review]

**Comment by rafaelbn** [comment]
Hi @hprasetyou , thanks you this PR. Please are you going to fix it? Thanks!

**Pull request closed without merging**